.github/workflows/trivy-scan.yml

```diff
@@ -21,8 +21,11 @@ jobs:
           egress-policy: audit
 
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      - name: Create Licenses Placeholder Directory
```

Comment:
this shouldn't be needed since licenses don't contain vulnerabilities, and adds a unneeded extra step